README.md

```diff
@@ -6,6 +6,7 @@ This is meant to be a custom cog to be added to a [Red DiscordBot](https://githu
 + Being a programmer is hard -> programmer, I barely know her!
 + In general Dad's jokes are amazing -> General Dad + a gif of a person saluting
 + Dad's jokes are a smashing good time -> A gif of Nigel Thornberries face plastered on someone elses body
++ One does not simply push one past Dad -> A gif from LOTR will serve as a simple reminder
```

Comment:
I appreciate so much that you added this part. Pro developer move.

Reply:
Gotta start somewhere, documentation is key